crates/cadenza-eval/src/lib.rs

```diff
@@ -41,9 +41,8 @@ pub use diagnostic::{
 pub use diagnostic::{Error, ErrorKind};
 pub use env::Env;
 pub use eval::{
-    builtin_add, builtin_div, builtin_eq, builtin_gt,
-    builtin_gte, builtin_lt, builtin_lte, builtin_mul, builtin_ne,
-    builtin_sub, eval,
+    builtin_add, builtin_div, builtin_eq, builtin_gt, builtin_gte, builtin_lt, builtin_lte,
+    builtin_mul, builtin_ne, builtin_sub, eval,
```

Comment:
All of these built-in operators need to be special forms as well, since they have both an evaluation implementation and then need to define how they are translated into IR.